includes/polling/netstats-snmp.inc.php

```diff
@@ -54,7 +54,7 @@
             $value = $data_array[0][$oid];
         }
         else {
-            $value = 'U';
+            $value = null;
```

Comment:
Sorry, I wasn't too clear.

In the rrd_update function which is called a little later, the following occurs:

```php
        foreach ($options as $k => $v) {
            if (!is_numeric($v)) {
                $v = U;
            }

            $values[] = $v;
        }
```

This sets U for non numeric data before inserting into the RRD. This bit of code here is doing that again, but on a specific subset of the data.

In my opinion, munging the data for inclusion into a datastore should belong in that datastores insert function.